### PR TITLE
LPS-190069

### PR DESCRIPTION
**Pull request by ces-locpham**
Hi @wanderlast ,
https://liferay.atlassian.net/browse/LPS-190069
Please help me review this PR.
I also updated the Vatican country follow your guidance and revert the countries's idd in the Caribbean Region.
If I missed from your ideas, feel free to comment it in here.
I am looking forward to hearing your response
Many thanks,
Loc

### CODE REVIEW

File: modules/apps/address/address-impl/src/main/resources/com/liferay/address/dependencies/countries.json [hunk 1]
```diff
@@ -1943,6 +1935,13 @@
 		"number": 548,
 		"zipRequired": false
 	},
+	{
+		"a2": "VA",
+		"a3": "VAT",
+		"name": "vatican-city",
```
Comment (wanderlast) [inline modules/apps/address/address-impl/src/main/resources/com/liferay/address/dependencies/countries.json]: Missing IDD, it should be "39" similar to how it was prior to the changes you made.

File: modules/apps/address/address-impl/src/main/resources/com/liferay/address/dependencies/countries.json [hunk 2]
```diff
@@ -219,9 +219,9 @@
 		"a2": "BQ",
 		"a3": "BES",
 		"idd": 599,
-		"name": "bonaire-st-eustatius-saba",
+		"name": "bonaire-sint-eustatius-saba",
```
Comment (wanderlast) [inline modules/apps/address/address-impl/src/main/resources/com/liferay/address/dependencies/countries.json]: I'm marking this as an example, but this applies to all "name" changes. 

The "Name" seen here is actually meant to be a key listed in portal-language-lang's Language.properties file. For example, you can search and find "country.bonaire-st-eustatius-saba". Therefore, if you want to make changes to the listed names of the countries, you cannot do it in this file, you must do it in the Language.properties and Language_en.properties files. Please make changes in those files if any names need to be changed.

File: modules/apps/address/address-impl/src/main/resources/com/liferay/address/dependencies/countries.json [hunk 3]
```diff
@@ -730,7 +746,7 @@
 	{
 		"a2": "GG",
 		"a3": "GGY",
-		"idd": 44,
+		"idd": "44-1481",
```
Comment (wanderlast) [inline modules/apps/address/address-impl/src/main/resources/com/liferay/address/dependencies/countries.json:725]: While an IDD can have both the country code and an area code, we only need the country code in our case. Despite being labeled as "IDD," it's listed as "Country Calling Code" on the Country Management screen so the area code is not necessary. This also applies to the Isle of Man and possibly other entries, so please take a look if you get a chance and remove those. Thanks!

**Comment by wanderlast** [comment]
Hi @locpham97, please see the comments I've left on this and let me know if any need further explanation. Thanks!

**Comment by ces-locpham** [comment]
Hi @wanderlast ,
Thanks for replying to me.
Could you check it and mention any missing information here?.
Many thanks.

**Review by wanderlast — CHANGES_REQUESTED** [review]

**Comment by wanderlast** [comment]
Hi @locpham97,

Sorry about that, forgot to finalize it. Can you see the three comments I've left?

**Comment by ces-locpham** [comment]
Hi @wanderlast ,
Thanks for leaving the comments.
I will fix it now.

File: modules/apps/address/address-impl/src/main/resources/com/liferay/address/dependencies/countries.json (hunk 2 above)
Comment (ces-locpham) [reply modules/apps/address/address-impl/src/main/resources/com/liferay/address/dependencies/countries.json]: the responsibility for the task is updated again about regions and countries json.  I will revert it but I do not update the language.properties

**Comment by ces-locpham** [comment]
Hi @wanderlast ,
I updated it.

**Comment by ces-locpham** [comment]
Hi @wanderlast ,
Next week, 
I am on vacation. Could you leave the comments here or continue the LPS?
Many thanks,
Loc

**Comment by wanderlast** [comment]
This LGTM.

**Comment by ces-locpham** [comment]
Hey @wanderlast ,
Could you help me to forward this PR?
Thanks,
Loc

**Comment by ces-locpham** [comment]
Hi @wanderlast ,
 Any update for this PR?
I'd appreciate it if you any feedback or updates.
Thanks,
Loc

**Comment by wanderlast** [comment]
Hi @locpham97,
I'm currently waiting on feedback from the GS team on the client's requirements. I'll let you know once I hear back from them.

**Comment by ces-locpham** [comment]
Hi @wanderlast ,
Any update for this PR?
I'd appreciate it if you any feedback or updates.
Thanks,
Loc

**Comment by wanderlast** [comment]
Hi @locpham97,

Since you're not changing names or adding other parameters, I think we should omit the changes to Macau and Hong Kong here. We can readd them in the LPS that adds the actual system setting since we don't want to change the default behavior for clients without alerting them.

So please do the following:
1. Remove the notes about "(see also separate country code entry under X)" for Taiwan, Macao and Hong Kong
2. Remove the country listings for Macao and Hong Kong (Taiwan is already there and can stay)

If you need help explaining this to the client/GS, please let me know. Otherwise this looks good to me. Thanks!

**Comment by ces-locpham** [comment]
HI @wanderlast , 
I will update this PR. Could you help me forward this PR?
Many thanks,
Loc

**Comment by ces-locpham** [comment]
I updated it

**Review by wanderlast — APPROVED** [review]

**Comment by ces-locpham** [comment]
Hi @wanderlast , 
I want to ask more for this ticket. Should the next plan for the ticket  be sent to the product team?

**Comment by wanderlast** [comment]
@locpham97 yes, I'll be forwarding it to the product team

**Comment by ces-locpham** [comment]
@wanderlast Thanks so much

**Comment by ces-locpham** [comment]
Hi @wanderlast,

Could you please send me the PR that you forwarded to the product team? I'd like to track it for convenience.

**Comment by wanderlast** [comment]
Forwarded it to: https://github.com/liferay-commerce/liferay-portal/pull/4359

**Pull request closed without merging**